torchinductor/ir.py

```diff
@@ -2024,12 +2024,19 @@ def create(
                 V.graph.sizevars.guard_static_shape(output_size[-1])
             )
 
+        # memory layout order of output depends on x
+        # symbolic stride of x, e.g. [s1**2*s0, 1, s0*s1, s1]
+        # order according to .atoms(), but constant "1" < "s1"
+        x_stride_order = [
+            len(expr.atoms()) - int(expr.is_number) for expr in x.get_stride()
```

Comment:
It might be better to use `V.graph.sizevars.size_hint`.

There is some similar stuff (for a memory address, not a list of strides) here:
https://github.com/pytorch/torchdynamo/blob/488fad00afc7e2a1e11adc9b13587aa25466f5b4/torchinductor/sizevars.py#L232

Reply:
Use `V.graph.sizevarssize_hint` to get actual strides. Thanks